components/RCLink/RCLink.tsx

```diff
@@ -0,0 +1,26 @@
+import { Link as DSLink } from "@nypl/design-system-react-components"
+import Link from "next/link"
+import type { ReactNode } from "react"
+
+interface RCLinkProps {
+  active?: boolean
+  href: string
+  children: ReactNode
+}
+
+/**
+ * The RCLink component is a utility that wraps the design system Link component with
+ * Next's Link component to allow for correct navigation in Next per the design system's
+ * docs. It also includes an 'active' prop used for styling the SubNav component.
+ */
+const RCLink = ({ href, children, active = false, ...rest }: RCLinkProps) => {
+  return (
+    <Link href={href} passHref {...rest}>
+      <DSLink fontWeight={active && "var(--nypl-fontWeights-bold)"}>
```

Comment:
Use design tokens in jsx files and css vars in the css module files. In this case, this can be `"bold"` instead of `"var(--nypl-fontweights-bold)"`.